### DISCUSSION TITLE
Is it possible to create custom conversions?

### DISCUSSION THREAD

**Discussion by Nizio-Dev** [riok/mapperly#1573] [Q&A]
Hello, I have a use case where I need to map IDataReader from ADO.NET to my DTO. Is there a way to create a custom converter?
(It would need to take source (IDataReader) and a destination property)

**Comment by latonz** [comment]
You can either use a [user-implemented mapping method](https://mapperly.riok.app/docs/configuration/user-implemented-methods/) if you want to apply the "converter" to all mappings of the given types, for a single property you can use a [user-implemented property mapping](https://mapperly.riok.app/docs/configuration/mapper/#user-implemented-property-mappings).

**Reply by Nizio-Dev** [reply]
Is it possible to get destination property from it somehow?

**Reply by latonz** [reply]
Nope, then you would need to opt for [before-/aftermap](https://mapperly.riok.app/docs/configuration/before-after-map/) and probably ignore the property.
